Softeq.NetKit.Components.EventBus.Service/Connection/ServiceBusPersisterConnection.cs

```diff
@@ -42,6 +43,19 @@ public MessageReceiver CreateMessageReceiver(string entity)
             }
         }
 
+        public async ValueTask DisposeAsync()
```

Comment:
The whole class looks like it is not intended to be derived. The dispose method is only one too. Maybe this class could be made sealed to explicitly outline this?